core/jvm/src/main/scala/me/shadaj/scalapy/interpreter/CPythonAPI.scala

```diff
@@ -1,9 +1,9 @@
-package me.shadaj.scalapy.py
+package me.shadaj.scalapy.interpreter
 
 import com.sun.jna.{Native, NativeLong, Memory}
 
 class CPythonAPIInterface {
-  Native.register("python3.7m")
+  Native.register("python3")
```

Comment:
FYI, I gave this branch a try locally (to check if it would address issues I've been having with numpy, on macOS). It seems conda doesn't create `libpython3.dylib` links in its `lib` directory, only things like `libpython3.8.dylib`. You may want to try `Native.register("python3.8")` (no `m` suffix for me for `3.8`, unlike for `3.7`) if the call to `Native.register("python3")` fails.

Reply:
Or maybe allow users to configure the library name?

Reply:
Hmm yeah, I think probably the best option is to have an environment variable that the user can set to override the default Python version and also have a fallback to `python3.8` if `python3` fails to load.